modules/main/extra-modules.conf

```diff
@@ -18,6 +18,8 @@ algif_skcipher
 cryptomgr
 dm_crypt
 dm_mod
+dm_verity
+overlay
```

Comment:
Could you please provide some more information on why we need this?  Is this expected to be used only for the CVM use case?

Reply:
Yes, the CVM use case is the driver for this. The change is described in spec FO164.